packages/material-ui/src/Breadcrumbs/BreadcrumbCollapsed.test.js

```diff
@@ -26,6 +27,25 @@ describe('<BreadcrumbCollapsed />', () => {
     assert.strictEqual(wrapper.hasClass(classes.root), true);
   });
 
+  it('should be focusable', () => {
+    const wrapper = shallow(<BreadcrumbCollapsed />);
```

Comment:
We are moving away from shallow tests, please prefer the mount API.

Reply:
Just interesting. 
Why you move from shallow to mount ? Because of children?

Reply:
The higher the level of abstraction the test can be the better. From our experience, shallow tests didn't bring much to the table. Using mount and accessibility based assertions improves the quality of the implementation and enable more refactoring.